### PR TITLE
Create Garvitaj12.json

### PR DESCRIPTION
**Pull request by Garvitaj12**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6444"></a>

### CODE REVIEW

File: data/Garvitaj12.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Garvita Jain",
+  "type": "personal",
+  "bio": "B.Tech student",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
```
Comment (jaydeepkhatri) [inline data/Garvitaj12.json:7]: The links & icon are for Github, meanwhile you have written "Follow me on Twitter"

**Review by jaydeepkhatri — CHANGES_REQUESTED** [review]
Hi @Garvitaj12,
Thank you for contributing to Linkfree.

I have found 1 potential small change.
Thanks :)

**Review by harshsingh32 — COMMENTED** [review]
looks fine require some changes

File: data/Garvitaj12.json (hunk 1 above)
Comment (SaraJaoude) [reply data/Garvitaj12.json:7]: > The links & icon are for Github, meanwhile you have written "Follow me on Twitter"

Given the length of time this PR has been open and as this is not a blocker, I will merge this.  If you wish to make changes to your Profile then I suggest you have a look at our Editing Guide in Docs: https://linkfree.io/docs/how-to-guides/editing

**Pull request merged**